### DISCUSSION TITLE
Run directly train.py

### DISCUSSION THREAD

**Discussion by Oleg-1978** [ultralytics/ultralytics#2646] [Q&A]
https://github.com/ultralytics/ultralytics/tree/main/ultralytics/yolo/v8/detect/

In yolo5 I can run train.py directly.
Now in Yolo8 I have problem to run https://github.com/ultralytics/ultralytics/tree/main/ultralytics/yolo/v8/detect/train.py.
It's normal or not?

**Comment by glenn-jocher** [comment]
@Oleg-1978 hello there! 👋

In YOLOv8, the training process is integrated into a streamlined command-line interface, and the structure is a bit different from YOLOv5. If you're encountering issues running `train.py` directly from the path you mentioned, it's likely because the entry point for training has changed.

For YOLOv8, you should use the CLI commands as outlined in our documentation. The training process is initiated through the CLI, which handles the setup and execution of the training routine. Please refer to the Train section in our documentation for guidance on how to properly train your YOLOv8 models.

If you're following the correct procedure and still facing issues, please provide more details about the error messages or problems you're encountering, and we'll be happy to assist you further! 😊

Happy training!
